### ISSUE TITLE
Accessibility Audit Results - many "todo"s

### ISSUE THREAD

**Issue by russtuck** [gordon-cs/gordon-360-ui#653]
CPS 353 students did accessibility audits of various parts of 360.gordon.edu, and wrote up findings and recommendations.  They are stored in this Drive folder: 
https://drive.google.com/drive/folders/1kGo5dgE9_SYtCJP0b8YOPdXeilRjnSah?usp=sharing

**Comment by bennettforkner** [comment]
These are not going to get addressed in this format and they are 4 years old. If there are accessibility issues, they should be documented in GitHub so that they can be tracked individually rather than as a link to a folder of 16 files for the developers to search through.

**Comment by EjPlatzer** [comment]
Reopening because this is a significant resource for improving the accessibility of the site, even if it isn't the most convenient format. If nothing else, it can serve as a helpful starting point to catalog known issues, including possible resolutions here in GitHub.

**Comment by bennettforkner** [comment]
Sounds good to me! (after conversations offline :) )
